content/02.intro.md

```diff
@@ -1,10 +1,10 @@
 ## Introduction
 
-Deep learning (DL), a subfield of machine learning (ML) that focuses on deep artificial neural networks, is exploding in popularity and is increasingly being used for biological data analysis [@doi:10.1098/rsif.2017.0387].
-However, DL itself remains an active area of research, and the complexity of this field poses a large barrier of entry to those who would like to correctly utilize state-of-the-art DL technology in biological research applications.
-While much has been written about the impressive results of DL in biology, there is a comparative dearth of literature articulating best practices for its use, with most instructional literature focusing on ML, rather than DL [@doi:10.1186/s13040-017-0155-3].
+Deep learning (DL), a subfield of machine learning (ML) implementing hierarchical artificial neural networks, is increasingly used for the characterization of genomic and biological data [@doi:10.1098/rsif.2017.0387].
```

Comment:
"Characterization" is often used in biology literature, but it may be better to keep it at the more common vernacular since these are tips that hopefully cross fields. 

I would just say biological data. This encompasses genomic data but does not limit the applicability of these tips to just genomic data.

Reply:
I would also favor "analysis" because characterization may be more like "describing" than "predicting" (usually ML/DL methods are used for various prediction tasks in bio as far as I know). E.g., in a drug discovery context, I am thinking of DL for predicting active molecules as drug candidates, and molecular dynamics simulations and wet-lab experiments would then be used to characterize the binding mechanism or sth like that. Not sure about genomics-specific work though